src/admin/settings/settings.html

```diff
@@ -0,0 +1,126 @@
+<div ng-if="!platformSettings.readOnly" class="form-group settings-box-medium">
+    <div class="settings-box-content">
+        <legend class="frame-title settings-box-header">{{'admin.settings.upload' | translate}}</legend>
+        <div class="settings-form">
+            <legend>{{'admin.platform.settings' | translate}}</legend>
+            <form class="form-inline inside" id="settingsFileForm" action="module/admin/api/settings/platform/upload" method="post" enctype="multipart/form-data">
+                <fieldset>
+                    <label class="control-label input-label">{{'admin.settings.fileChosen' | translate}}</label>
+                    <div class="btn-group">
+                        <span class="btn btn-default">
+                            <span>{{'server.button.selectFile' | translate}}</span>
+                            <input type="file" id="fileInput" name="file"/>
+                        </span>
+                    </div>
+                    <span class="btn btn-primary" ng-click="uploadSettings()">{{'admin.submit' | translate}}</span>
+                </fieldset>
+            </form>
+        </div>
+    </div>
+</div>
+<div ng-if="!platformSettings.readOnly" class="form-group settings-box-small">
+    <div class="settings-box-content">
+        <legend class="frame-title settings-box-header">{{'admin.settings.location' | translate}}</legend>
```

Comment:
The `<legend>` element only adds value to a page within a fieldset -- for every legend on this page, you should either change it to a heading element (h3 or h4) OR move it inside a fieldset (which is two lines below each item) ie:
```
<form>
    <fieldset>
        <legend>What the grouped fields do</legend>
        ...Add inputs and things....
   </fieldset>
</form>
```